ui/src/components/CERN_FOOTER.css

```diff
@@ -22,22 +22,32 @@
   max-width: 1140px;
   width: 100%;
   margin: auto;
+  padding: 55px 3%;
   background: #191919;
 }
 
+/*
 #cern-footer .row {
-  -webkit-text-size-adjust: 100%;
-  -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
-  line-height: 1.428571429;
+    -webkit-text-size-adjust: 100%;
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    line-height: 1.428571429;
+}*/
+
+#cern-footer .ant-menu {
+  width: auto;
+}
+
+#cern-footer .ant-menu-item {
   font: normal normal normal 16px/28px Source Sans Pro Regular;
-  margin: auto;
+  width: 200px;
+  left: 125px;
 }
 
-.row:before,
+/* .row:before,
 .row:after {
-  display: table;
-  content: " ";
-}
+    display: table;
+    content: " ";
+} */
```

Comment:
just delete them don't comment, if we want to recover it we can always go back to git history